src/utils/paymentQrValidation.test.ts

```diff
@@ -1,77 +1,18 @@
 
-    expect(parseAndValidateSGQR(sgqrPayload)).toStrictEqual({
-      payloadFormatIndicator: "01",
-      pointOfInitiationMethod: "11",
-      merchantAccountInformation: {
-        amex: ["1234567890", "312345678901234"],
-        mastercard: ["512345678912345"],
-        nets: {
-          globallyUniqueIdentifier: "SG.COM.NETS",
-          version: "0",
-          qrIssuerUen: "2014018328",
-          qrExpiryTimestamp: "311288235900",
-          merchantId: "000111870324000",
-          terminalId: "88587201",
-          editableTxnAmountIndicator: true,
-          qrType: undefined,
-          signature: "4E5DC3D8",
-        },
-        paynow: {
-          globallyUniqueIdentifier: "SG.PAYNOW",
-          proxyType: "U",
-          proxyValue: "+621234567890123",
-          editableTxnAmountIndicator: false,
-          reference: "1234567R",
-          qrExpiryDate: "20991231",
-        },
-        unionpay: ["2500034400010344500034453110001"],
-        visa: ["4761360000000*17"],
-      },
-      sgqrIdentityInformation: {
-        uniqueIdentifier: "SG.SGQR",
-        sgqrIdNumber: "180307510317",
-        version: "01.0003",
-        postalCode: "081006",
-        levelNumber: "02",
-        unitNumber: "138",
-        miscellaneous: "Counter01",
-        newVersionDate: "20180407",
-      },
-      merchantCategoryCode: "5814",
-      transactionCurrency: "702",
-      transactionAmount: undefined,
-      tipOrConvenienceIndicator: undefined,
-      valueOfConvenienceFeeFixed: undefined,
-      valueOfConvenienceFeePercentage: undefined,
-      countryCode: "SG",
-      merchantName: "FOOD XYZ PTE LTD",
-      merchantCity: "SINGAPORE",
-      postalCode: "081006",
-      additionalDataFieldTemplates: {
-        billNumber: "1234567R",
-        mobileNumber: undefined,
-        storeLabel: undefined,
-        loyaltyNumber: undefined,
-        referenceLabel: undefined,
-        customerLabel: "8765430",
-        terminalLabel: undefined,
-        purposeOfTransaction: undefined,
-        additionalConsumerDataRequest: undefined,
-        paymentSystemSpecificTemplates: {
-          paynow: undefined,
-          nets: undefined,
-        },
-      },
-      merchantInformationLanguageTemplate: undefined,
-      crc: "5E8D",
-    });
+    expect(paymentQrValidate(sgqrPayload)).toBe(true);
+  });
+
+  it("should retrun false if invalid qr payload", () => {
+    expect.hasAssertions();
```

Comment:
can change all the `expect.hasAssertions()` to `expect.assertions(<number>)`!

Reply:
updated with `assertions`